addons/visibility/XEH_postInit.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+
+["ZEN_displayCuratorLoad", {
+    if (GVAR(enabled)) then {
+        call FUNC(start);
+    };
+}] call CBA_fnc_addEventHandler;
+
+["ZEN_displayCuratorUnload", {
+    call FUNC(stop);
+}] call CBA_fnc_addEventHandler;
+
+["CBA_SettingChanged", {
+    params ["_setting", "_value"];
+    if (_setting isEqualTo QGVAR(enabled)) then {
+        if (isNull (findDisplay IDD_RSCDISPLAYCURATOR)) exitWith {};
```

Comment:
I think this can be moved to the top.

Reply:
Wouldn't the setting name check be faster?

Reply:
you could make both of them an exitWith.
I  think that would look nicer.

Reply:
Ah right, forgot that existed. I'll switch to that